Index: app/src/main/res/layout/activity_main.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/layout/activity_main.xml b/app/src/main/res/layout/activity_main.xml
--- a/app/src/main/res/layout/activity_main.xml	
+++ b/app/src/main/res/layout/activity_main.xml	(date 1777287927143)
@@ -143,6 +143,7 @@
             android:backgroundTint="#222222"
             android:fontFamily="@font/dot_digital7"
             android:text="9"
+            android:textColorLink="#FFFFFF"
             android:textSize="44sp"
             android:textStyle="bold"
             app:cornerRadius="100dp" />
Index: app/src/main/java/com/affiocutuli/calcolatrice/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/affiocutuli/calcolatrice/MainActivity.java b/app/src/main/java/com/affiocutuli/calcolatrice/MainActivity.java
--- a/app/src/main/java/com/affiocutuli/calcolatrice/MainActivity.java	
+++ b/app/src/main/java/com/affiocutuli/calcolatrice/MainActivity.java	(date 1776759539028)
@@ -80,12 +80,10 @@
             }
         });
 
-
         bRadice.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                if(tvInput.getText().toString().isEmpty())
-                {
+                if (tvInput.getText().toString().isEmpty()) {
                     return;
                 }
                 num1 = Float.parseFloat(tvInput.getText().toString());
@@ -99,11 +97,10 @@
         bPotenza.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                if(tvInput.getText().toString().isEmpty())
-                {
+                if (tvInput.getText().toString().isEmpty()) {
                     return;
                 }
-                num1=Float.parseFloat(tvInput.getText().toString());
+                num1 = Float.parseFloat(tvInput.getText().toString());
                 operazione = "^";
                 tvStorico.setText(tvInput.getText().toString() + " ^");
                 tvInput.setText("");
@@ -113,8 +110,7 @@
         bPercento.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                if(tvInput.getText().toString().isEmpty())
-                {
+                if (tvInput.getText().toString().isEmpty()) {
                     return;
                 }
                 num1 = Float.parseFloat(tvInput.getText().toString());
@@ -140,10 +136,8 @@
             @Override
             public void onClick(View view) {
                 String attuale = tvInput.getText().toString();
-                if (!attuale.contains("."))
-                {
-                    if (attuale.isEmpty())
-                    {
+                if (!attuale.contains(".")) {
+                    if (attuale.isEmpty()) {
                         tvInput.setText("0.");
                     } else {
                         tvInput.setText(attuale + ".");
@@ -165,11 +159,11 @@
         });
 
         b1.setOnClickListener(new View.OnClickListener() {
-                                  @Override
-                                  public void onClick(View view) {
-                                      numeroClick(view);
-                                  }
-                              });
+            @Override
+            public void onClick(View view) {
+                numeroClick(view);
+            }
+        });
 
         b2.setOnClickListener(new View.OnClickListener() {
             @Override
@@ -238,6 +232,7 @@
         Button b = (Button) view;
         tvInput.append(b.getText().toString());
     }
+
     private void preparaOperatore(String op) {
         if (!tvInput.getText().toString().isEmpty()) {
             num1 = Float.parseFloat(tvInput.getText().toString());
